### ISSUE TITLE
Failure of RVM installation not causing chef-client to fail

### ISSUE THREAD

**Issue by finack** [martinisoft/chef-rvm#182]
If retrieval of the rvm install script fails, it does not cause the failure to surface to Chef and abort the client.

The command in question is (for me):

```bash
 https://get.rvm.io | bash  -s -- --version 1.18.14
```

I believe the issue is that it fails (because of rvm.io ssl error) but bash is returning exit code 0.

Maybe this should be two steps? One curl the script, two run the script?

### Background

Currently there is an issue with the SSL Certificate on rvm.io. Thanks to @cromulus comment in https://github.com/fnichol/chef-rvm/issues/51, I was able to fix the issue by changing the rvm install path to:

```ruby
node['rvm']: ['installer_url']="https://raw.github.com/wayneeseguin/rvm/master/binscripts/rvm-installer"
```
For those searching for this problem, the error I received in my chef logs was

```
private method `chomp' called for nil:NilClass
```

See https://gist.github.com/finack/5174496 for my logs.

**Comment by noahgibbs** [comment]
I'm seeing a failure in Ruby installation, too. When Ruby install fails, it's not aborting the chef run. It only prints a warning saying that the Ruby wasn't installed.

**Comment by cmluciano** [comment]
This should be fixed for system with #285 and user in #298. Please reopen if this is still an issue
